src/publish/performPublishOverrides.ts

```diff
@@ -0,0 +1,55 @@
+import { PackageInfos } from '../types/PackageInfo';
+import * as fs from 'fs-extra';
+
+export const acceptedKeys = ['types', 'typings', 'main', 'module', 'exports', 'repository', 'bin', 'browser', 'files'];
+
+export function performPublishOverrides(packagesToPublish: string[], packageInfos: PackageInfos) {
+  for (const pkgName of packagesToPublish) {
+    const info = packageInfos[pkgName];
+    const packageJson = fs.readJSONSync(info.packageJsonPath);
```

Comment:
Now that this function performs 2 separate actions, we should really document / separate it out so it's clearer what is done to the package.json files.